src/utils/PokeApi.jsx

```diff
@@ -0,0 +1,62 @@
+export const baseUrl = "https://pokeapi.co/api/v2/pokemon";
+
+export async function fetchPokemon() {
+  function arrangeTypes(data) {
+    const typeHolder = document.getElementById("pokeType");
+    const pokeTypes = data.types;
+    if (pokeTypes[1]) {
+      typeHolder.textContent =
+        pokeTypes[0].type.name + " " + pokeTypes[1].type.name;
+    } else {
+      return (typeHolder.innerText = pokeTypes[0].type.name);
+    }
+  }
+
+  try {
+    const pokemonName = document
+      .getElementById("pokemonName")
```

Comment:
`document.querySelector` or `getElementById` should **not be used** in `React` because it’s a **very slow** operation for the browser. React was specially created not to search anything in the DOM. So, you can directly operate with the markup inside the components. Or you can use hook `useRef` for the reference.


But you should `pass` the name of the pokemon into `fetchPokemon`